src/app/paging/paging-demo.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
```

Comment:
This isn't necessarily something we need to keep, but I was using it to verify the public_api export.